### PR TITLE
Update two constructor tests of Array.fromAsync

### PR DESCRIPTION
**Pull request by rmahdav**
Based on [this fix](https://github.com/tc39/proposal-array-from-async/pull/41) on Array.fromAsync spec to avoid calling constructor twice, the following two tests needed to be updated:
built-ins/Array/fromAsync/this-constructor
built-ins/Array/fromAsync/this-constructor-operations

### CODE REVIEW

File: test/built-ins/Array/fromAsync/this-constructor-operations.js [hunk 1]
```diff
@@ -60,7 +60,6 @@ asyncTest(async function () {
 
   // Note https://github.com/tc39/proposal-array-from-async/issues/35
```
Comment (ptomato) [inline test/built-ins/Array/fromAsync/this-constructor-operations.js:61]: We can delete this comment now, too, since the fix makes it obsolete.

**Review by ptomato — APPROVED** [review]
Thanks for the contribution! This looks good. Would you mind removing the comment as well before we merge it in?

File: test/built-ins/Array/fromAsync/this-constructor-operations.js (hunk 1 above)
Comment (rmahdav) [reply test/built-ins/Array/fromAsync/this-constructor-operations.js]: Done. Thanks for pointing that out.

**Review by ptomato — APPROVED** [review]
Thanks!

**Pull request merged**